src/config/styles/theme.ts

```diff
@@ -45,6 +46,7 @@ const darkTheme = {
     red: colors.red,
     redHover: colors.redHover,
     boxShadowPrimary: colors.boxShadowPrimary,
+    cardWhite: colors.cardWhite,
```

Comment:
This doesnt look to well in dark mode, suggesting to change it

Reply:
Resolved in 385038422e93b2180199ee15ab9b66793d3ad92c